### PR TITLE
Entries: Document "skipped" in testDone details object

### LINKED ISSUE #107: Document "skipped" in testDone details object
Introduced in https://github.com/jquery/qunit/commit/3f08a1aa1e03ec99eed7ebe31b36132ccc245a56 (1.16.0).

### PR DESCRIPTION
**Pull request by Arkni**
Fixes #107

### CODE REVIEW

File: entries/QUnit.testDone.xml [hunk 1]
```diff
@@ -26,14 +26,17 @@
...
+			</property>
 		</argument>
 	</signature>
 	<desc>Register a callback to fire whenever a test ends.</desc>
 	<example>
 		<desc>Register a callback that logs results of a single test</desc>
 		<code><![CDATA[
 QUnit.testDone(function( details ) {
-	console.log( "Finished running: ", details.module, details.name, "Failed/total: ", details.failed, details.total, details.duration );
+	console.log( "Finished running: ", details.module, details.name, "Failed/total: ", details.failed, details.total, details.duration, details.skipped );
```
Comment (platinumazure) [inline entries/QUnit.testDone.xml]: Not sure about this example- the runtime and skip count both just show up after the failed/total numbers (which are annotated).

Comment (Arkni) [inline entries/QUnit.testDone.xml]: `details.skipped` here is a Boolean which indicates whether the test was skipped or not. But, I'm open to suggestion to change this test to something better.

**Comment by platinumazure** [comment]
I was just complaining about the lack of label. If someone used this line,
it'd just have a true/false at the end without any context. The same
applies to the duration though.

**Comment by Arkni** [comment]
Sorry for the misunderstanding. I will change it to something more explicit like:
```js
QUnit.testDone( function( details ) {
  var result = {
    "Module name": details.module,
    "Test name": details.name,
    "Assertions": {
      "Total": details.total,
      "Passed": details.passed,
      "Failed": details.failed
    },
    "Runtime": details.runtime
  };

  console.log( JSON.stringify( result, null, 2 ) );
} );
```

**Comment by platinumazure** [comment]
That would be way awesome, thanks! Just don't forget to include the skipped
count. Thanks for bearing with my miscommunication.

**Comment by Arkni** [comment]
Done!

File: entries/QUnit.testDone.xml [hunk 2]
```diff
@@ -26,15 +26,30 @@
 			<property name="runtime" type="Number">
 				<desc>The total runtime in millseconds of the test, including beforeEach and afterEach</desc>
 			</property>
+			<property name="skipped" type="Boolean">
+				<desc>Indicate whether or not the current test was skipped</desc>
```
Comment (trentmwillis) [inline entries/QUnit.testDone.xml]: Super tiny nitpick: should be `Indicates` with an `s`

Comment (Arkni) [inline entries/QUnit.testDone.xml]: Ops my bad.
Fixed!

**Comment by trentmwillis** [comment]
Thanks!

**Pull request merged**